src/Microsoft.AspNet.OData.Shared/DeltaOfTStructuralType.cs

```diff
@@ -89,12 +111,13 @@ public Delta(Type structuralType, IEnumerable<string> updatableProperties)
         /// names, including those of dynamic properties, are ignored.</param>
         /// <param name="dynamicDictionaryPropertyInfo">The property info that is used as dictionary of dynamic
         /// properties. <c>null</c> means this entity type is not open.</param>
+        /// <param name="isComplexType">Boolean value to determine if its a complex type</param>
```

Comment:
We can't use `structuralType` properties or supported methods to check if it's a complex type instead of introducing this parameter?